examples/00-scaleOverlay/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE HTML>
```

Comment:
I'm a little torn. On one-hand simple, granular examples are easier to learn from to use a tool. On the other hand, a lot of simple examples can become hard to maintain. For the time being, until we get things hammered out, I think we should slowly add on to the "global" example, and delete the ones that are no longer applicable.

When this becomes the official branch, we can double back and rethink how best to create and maintain examples as a form of documentation.

Reply:
If you'd like, we can merge with this, and I can use it as a guide to update the primary example, and then delete it? I've also wanted to add a simple way to bind to different mouse buttons in the main example so you can test multiple tools in tandem.

Reply:
hmm, make sense. 
I will figure out a way to add this tool into the global example and delete this example.